doc/try_into.md

```diff
@@ -0,0 +1,155 @@
+% What #[derive(TryInto)] generates
+
+This derive allows you to convert enum variants into their corresponding
+variant types.
+One thing to note is that this derive doesn't actually generate an
+implementation for the `TryInto` trait.
```

Comment:
Can you note somewhere here that this derive only works on the `nightly` compiler?